src/styles/custom-components.css

```diff
@@ -130,6 +96,11 @@
     right: var(--su2);
 }
 
+.s-btn-group--radio .s-btn.s-editor-btn {
```

Comment:
I understand you want to change `.s-btn-group--radio` with `.s-btn-group__radio` as part of [this PR](https://github.com/StackExchange/Stacks/pull/1346). For now though this is correct.